app/services/decline_offer.rb

```diff
@@ -12,7 +12,7 @@ def save!
       StateChangeNotifier.new(:declined, @application_choice).application_outcome_notification
     end
 
-    NotificationsList.for(@application_choice).each do |provider_user|
+    NotificationsList.for(@application_choice, include_ratifying_provider: true).each do |provider_user|
```

Comment:
The card mentions we care about declined offers – do we care about decline_by_default ones?